packages/webdriver/protocol/appium.json

```diff
@@ -748,7 +748,7 @@
       "ref": "http://appium.io/docs/en/commands/device/app/background-app/",
       "parameters": [{
         "name": "duration",
-        "type": "number",
+        "type": "(number|null)",
```

Comment:
Where is this documented that it also accepts `null`?

Reply:
See parameter description below:
> timeout to restore app, if 'null' app won't be restored